### PR TITLE
Added brief summary about Troposphere

### PR DESCRIPTION
**Pull request by abdulirfan3**
Added brief summary about Troposphere under CloudFormation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
 ### CloudFormation Tips
 
 -	[Troposphere](https://github.com/cloudtools/troposphere) is a Python library that makes it much easier to create CloudFormation templates.
+	- Currently supports AWS and OpenStack resource types.
+	- Does not support all of the AWS resource type that can be built using pure CloudFormation/JSON.
```
Comment (ThanosBaskous) [inline README.md]: Grammar suggestion: "Does not support every AWS resource type that can be built using pure CloudFormation/JSON."

File: README.md [hunk 2]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
...
+	- Built in error checking.
+	- Recommended soft dependency is awacs(https://github.com/cloudtools/awacs), which allows you to generate AWS access policy in JSON by writing python code.
```
Comment (ThanosBaskous) [inline README.md]: Grammar: "...which allows you to generate AWS access policy in JSON..." -> "...which allows you to generate AWS access policies in JSON..."

Comment (ThanosBaskous) [inline README.md]: Phrasing/brevity suggestion: "...which allows you to generate AWS access policy in JSON by writing python code." -> "...which allows you to generate AWS access policies in JSON using python code."

Comment (ThanosBaskous) [inline README.md]: Please make "awacs" into a proper markdown link.

`[awacs](https://github.com/cloudtools/awacs)`

File: README.md (hunk 1 above)
Comment (jlevy) [inline README.md]: Also, does this apply to YAML too now? Clarify if so or if not?
Any link?

Comment (jlevy) [inline README.md]: Also, does this apply to YAML too now? Clarify if so or if not?
Any link?

File: README.md [hunk 3]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
 ### CloudFormation Tips
 
 -	[Troposphere](https://github.com/cloudtools/troposphere) is a Python library that makes it much easier to create CloudFormation templates.
+	- Currently supports AWS and OpenStack resource types.
+	- Does not support all of the AWS resource type that can be built using pure CloudFormation/JSON.
+	- Built in error checking.
```
Comment (jlevy) [inline README.md]: This point could be more concrete or have a link to Troposphere docs or similar? By itself don't know what this really means.

File: README.md [hunk 4]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
 ### CloudFormation Tips
 
 -	[Troposphere](https://github.com/cloudtools/troposphere) is a Python library that makes it much easier to create CloudFormation templates.
+	- Currently supports AWS and OpenStack resource types.
```
Comment (jlevy) [inline README.md]: As usual, a link is best.

File: README.md [hunk 5]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
 ### CloudFormation Tips
 
 -	[Troposphere](https://github.com/cloudtools/troposphere) is a Python library that makes it much easier to create CloudFormation templates.
+	- Currently supports [AWS](https://github.com/cloudtools/troposphere#currently-supported-aws-resource-types) and [OpenStack](https://github.com/cloudtools/troposphere#currently-supported-openstack-resource-types) resource types.
+	- Not all AWS resource are supported that can be built using direct JSON/YAML CloudFormation Template
+	- Built in [error](https://github.com/cloudtools/troposphere#examples-of-the-error-checking-full-tracebacks-removed-for-clarity) checking.
+	- Recommended soft dependency is [awacs](https://github.com/cloudtools/awacs), which allows you to generate AWS access policy in JSON by writing python code.
```
Comment (jlevy) [inline README.md]: Nit: python -> Python
Nit: "A recommended soft ..."

File: README.md [hunk 6]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
 ### CloudFormation Tips
 
 -	[Troposphere](https://github.com/cloudtools/troposphere) is a Python library that makes it much easier to create CloudFormation templates.
+	- Currently supports [AWS](https://github.com/cloudtools/troposphere#currently-supported-aws-resource-types) and [OpenStack](https://github.com/cloudtools/troposphere#currently-supported-openstack-resource-types) resource types.
+	- Not all AWS resource are supported that can be built using direct JSON/YAML CloudFormation Template
```
Comment (jlevy) [inline README.md]: Nit: Missing period.

How about: Troposphere does not support all of the resources types you can describe with CloudFormation Templates.

File: README.md [hunk 7]
```diff
@@ -1128,6 +1128,10 @@ CloudFormation
 ### CloudFormation Tips
 
 -	[Troposphere](https://github.com/cloudtools/troposphere) is a Python library that makes it much easier to create CloudFormation templates.
+	- Currently supports [AWS](https://github.com/cloudtools/troposphere#currently-supported-aws-resource-types) and [OpenStack](https://github.com/cloudtools/troposphere#currently-supported-openstack-resource-types) resource types.
+	- Troposphere does not support all of the resources types you can describe with CloudFormation Templates.
```
Comment (ThanosBaskous) [inline README.md]: Nit/consistency: "CloudFormation Templates" -> "CloudFormation templates"

**Comment by jlevy** [comment]
LGTM, just that one last change? :)

**Comment by ThanosBaskous** [comment]
Thank you @abdulirfan3!

**Comment by jlevy** [comment]
Thanks @abdulirfan3 !

**Pull request merged**